rust/src/treefile.rs

```diff
@@ -688,6 +698,9 @@ struct TreeComposeConfig {
     #[serde(rename = "automatic-version-prefix")]
     automatic_version_prefix: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(rename = "version-suffix")]
```

Comment:
/bikeshed

Hmm actually, WDYT about `automatic-version-suffix` to match `automatic-version-prefix` since they work together? (Esp. since the "automatic" part is all in the suffix really).

Reply:
Sure, done.